core/o11y/src/lib.rs

```diff
@@ -21,7 +21,7 @@ pub const DEFAULT_RUST_LOG: &'static str = "tokio_reactor=info,\
 /// Once dropped, the subscriber is unregistered, and the output is flushed. Any messages output
 /// after this value is dropped will be delivered to a previously active subscriber, if any.
 #[allow(dead_code)] // Fields are never read
```

Comment:
I moved the lint annotation to the specific field that's currently unused. I don't particularly love `_fields` in structures due to them affecting the location at which the structure is constructed, which also tends to prevent use of shorthand field initialization.